internal/view/helm_history.go

```diff
@@ -56,6 +60,8 @@ func (h *History) bindKeys(aa ui.KeyActions) {
 
 	aa.Delete(ui.KeyShiftA, ui.KeyShiftN, tcell.KeyCtrlS, tcell.KeyCtrlSpace, ui.KeySpace, tcell.KeyCtrlD)
 	aa.Add(ui.KeyActions{
+		ui.KeyY:      ui.NewKeyAction(yamlAction, h.yamlCmd, true),
+		ui.KeyD:      ui.NewKeyAction("Describe", h.describeCmd, true),
```

Comment:
To stay on par with the helm chart parent view. I think we should have a 'Values' action. In case folks miss the 'enter' functionality??